src/Common/Core/Impl/Install/RInstallation.cs

```diff
@@ -123,33 +144,96 @@ public static class RInstallation {
             return path;
         }
 
-        public static void GoToRInstallPage() {
-            Process.Start("https://mran.revolutionanalytics.com/download");
+        public static Exception LaunchRClientSetup() {
+            var rClientExe = Path.Combine(Path.GetTempPath(), "RClientSetup.exe");
+            try {
+                using (var client = new WebClient()) {
+                    client.DownloadFileAsync(
+                        new Uri("http://go.microsoft.com/fwlink/?LinkId=800048", UriKind.Absolute), rClientExe);
+
+                    var startTime = DateTime.Now;
+                    while (client.IsBusy && (DateTime.Now - startTime).TotalMilliseconds < 5000) {
```

Comment:
Since it's called from UI thread when invoking command, this will basically block all UI for as long as it takes to download. I'd rather have this method return a task, and have `DownloadFileCompleted` event mark it as complete.